grakn-core/src/main/java/ai/grakn/graql/admin/Atomic.java

```diff
@@ -79,6 +80,12 @@
     default boolean isAllowedToFormRuleHead(){ return false; }
 
     /**
+     * @return error messages indicating ontological inconsistencies of this atomic
+     */
+    @CheckReturnValue
+    default Set<String> validateOntologically(){ return new HashSet<>();}
```

Comment:
It makes an atomic ontologically valid by default. I assume only resources and relations can be potentially invalid.